packages/api/lambdas/report-executions.js

```diff
@@ -0,0 +1,48 @@
+const get = require('lodash.get');
+
+const { getSnsEventMessageObject, isSnsEvent } = require('@cumulus/common/sns-event');
+// Temporarily change require while this module resides in the API package
+// const Execution = require('@cumulus/api/models/executions');
+const Execution = require('../models/executions');
+
+/**
+ * Create/update execution record from SNS message.
+ *
+ * @param {Object} executionRecord - An execution record
+ * @returns {Promise}
+ */
+async function createUpdateExecutionRecord(executionRecord) {
+  const executionModel = new Execution();
+  return executionModel.create(executionRecord);
```

Comment:
It is somewhat interesting to note that our model `create` methods actually do a PUT, meaning that they will create a new record for the included key or update the existing record for the included key. Thus, the code here will do what we expect, but it's a bit confusing to see a call to a method named `create` when it will also be used to update records.